apps/print/frontend/src/election_manager_app.tsx

```diff
@@ -1,16 +1,19 @@
 import { Redirect, Route, Switch } from 'react-router-dom';
 import { assertDefined } from '@votingworks/basics';
 
-import { getElectionDefinition } from './api';
+import { getElectionRecord } from './api';
 import { ScreenWrapper } from './components/screen_wrapper';
 import { PrintScreen } from './screens/print_screen';
 import { TitleBar } from './components/title_bar';
 import { electionManagerRoutes } from './routes';
 import { SettingsScreen } from './screens/settings_screen';
 
 function ElectionManagerElectionScreen(): JSX.Element | null {
-  const electionDefinitionQuery = getElectionDefinition.useQuery();
-  const { election } = assertDefined(electionDefinitionQuery.data);
+  const electionRecordQuery = getElectionRecord.useQuery();
+  if (!electionRecordQuery.isSuccess) return null;
```

Comment:
nit: I generally prefer the `then` clause of an `if` statement to be on its own line wrapped with braces even if trivial

Reply:
Agreed, slipped by!